### PR TITLE
映射文档 No.61

### PR DESCRIPTION
**Pull request by co63oc**
https://github.com/PaddlePaddle/docs/issues/5691

映射文档 No.61

torch.Tensor.repeat_interleave
torch.Tensor.sub 已存在

### CODE REVIEW

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.quantile.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+## [仅 torch 参数更多]torch.Tensor.quantile
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.quantile.md]: 分类名直接写`torch 参数更多 ` 即可~

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.quantile.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+### [torch.Tensor.quantile](https://pytorch.org/docs/1.13/generated/torch.Tensor.quantile.html#torch.Tensor.quantile)
+
+```python
+torch.Tensor.quantile(q, dim=None, keepdim=False, *, interpolation='linear')
+```
+
+### [paddle.quantile](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/Tensor_cn.html#quantile-q-axis-none-keepdim-false-name-none)
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.quantile.md]: 方法的签名可以写`paddle.Tensor.quantile`

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.quantile.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+```python
+paddle.Tensor.quantile(q, axis=None, keepdim=False, name=None)
+```
+
+其中 Paddle 相比 PyTorch 支持更多其他参数，具体如下：
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.quantile.md]: Pytorch的参数更多，写反了这里

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.quantile.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+### 参数映射
+|PyTorch|PaddlePaddle|备注|
+| ------- | ------- | ------- |
+|q|q|待计算的分位数。|
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.quantile.md]: 单元格内容前后可以敲几个空格，目前看起来有些拥挤，其他的参数映射部分也辛苦修改一下~

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.round.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
+## [仅 torch 参数更多]torch.Tensor.round
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.round.md]: torch 参数更多

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.round.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+torch.Tensor.round(decimals=0)
+```
+
+### [paddle.Tensor.round](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/Tensor_cn.html#round-name-none)
+
+```python
+paddle.Tensor.round(name=None)
+```
+
+其中 Paddle 相比 PyTorch 支持更多其他参数，具体如下：
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.round.md]: 这里写反了

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.round_.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
+## [仅 torch 参数更多]torch.Tensor.round_
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.round_.md]: torch 参数更多

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.round_.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+torch.Tensor.round_(decimals=0)
+```
+
+### [paddle.Tensor.round_](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/Tensor_cn.html#round-name-none)
+
+```python
+paddle.Tensor.round_(name=None)
+```
+
+其中 Paddle 相比 PyTorch 支持更多其他参数，具体如下：
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.round_.md]: 其中 PyTorch 相比 Paddle 支持更多其他参数，具体如下：

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.round_.md [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+
+#### decimals：要舍入到的小数位数
+```python
+# Pytorch 写法
+torch.tensor([3.345, 5.774]).round_(decimals=2)
+
+# Paddle 写法
+(paddle.to_tensor([3.345, 5.774]) * 1e2).round_() / 1e2
+
+# 注：Paddle 可使用 10 的 decimals 次方来实现
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.round_.md:31]: 另起一行加一个代码块的符号吧 ```

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.scatter.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+### [paddle.Tensor.put_along_axis](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/Tensor_cn.html#put-along-axis-arr-index-value-axis-reduce-assign)
+
+```python
+paddle.Tensor.put_along_axis(arr, index, value, axis, reduce="assign")
+
+```
+
+其中 Paddle 相比 PyTorch 支持更多其他参数，具体如下：
+
+### 参数映射
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.scatter.md:18]: 可以写一下 `arr` 参数应该如何设置呢

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.scatter_add.md [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+
+```python
+paddle.Tensor.put_along_axis(arr, index, value, axis, reduce="assign")
+
+```
+
+其中 Paddle 相比 PyTorch 支持更多其他参数，具体如下：
+
+### 参数映射
+|PyTorch|PaddlePaddle|备注|
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.scatter_add.md]: arr 参数的问题，同上

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.short.md [hunk 12]
```diff
@@ -0,0 +1,20 @@
+## [仅 torch 参数更多]torch.Tensor.short
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.short.md]: torch 参数更多

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.short.md [hunk 13]
```diff
@@ -0,0 +1,20 @@
...
+torch.Tensor.short(memory_format=torch.preserve_format)
+```
+
+### [paddle.Tensor.astype](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/Tensor_cn.html#put-along-axis-arr-index-value-axis-reduce-assign)
+
+```python
+paddle.Tensor.astype('int16')
+```
+
+其中 Paddle 相比 PyTorch 支持更多其他参数，具体如下：
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.short.md]: 其中 PyTorch 相比 Paddle 支持更多其他参数，具体如下：

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.size.md [hunk 14]
```diff
@@ -0,0 +1,30 @@
+## [仅 torch 参数更多]torch.Tensor.size
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.size.md]: torch 参数更多

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.size.md [hunk 15]
```diff
@@ -0,0 +1,30 @@
...
+```python
+paddle.Tensor.shape
+```
+
+其中 Paddle 相比 PyTorch 支持更多其他参数，具体如下：
+
+### 参数映射
+|PyTorch|PaddlePaddle|备注|
+| ------- | ------- | ------- |
+|dim|-|获取大小的轴|
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.size.md]: 备注可以改为`表示获取大小的轴，Paddle无此参数，需要进行转写。`

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.size.md [hunk 16]
```diff
@@ -0,0 +1,30 @@
...
+torch.Tensor.size(dim=None)
+```
+
+### [paddle.Tensor.shape](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/Tensor_cn.html#shape)
+
+```python
+paddle.Tensor.shape
+```
+
+其中 Paddle 相比 PyTorch 支持更多其他参数，具体如下：
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.size.md]: 其中 PyTorch 相比 Paddle 支持更多其他参数，具体如下：

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.stft.md [hunk 17]
```diff
@@ -0,0 +1,39 @@
+## [仅 torch 参数更多]torch.Tensor.stft
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.stft.md]: torch 参数更多

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.stft.md [hunk 18]
```diff
@@ -0,0 +1,39 @@
...
+torch.Tensor.stft(n_fft, hop_length=None, win_length=None, window=None, center=True, pad_mode='reflect', normalized=False, onesided=None, return_complex=None)
+```
+
+### [paddle.signal.stft](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/signal/stft_cn.html)
+
+```python
+paddle.signal.stft(x, n_fft, hop_length=None, win_length=None, window=None, center=True, pad_mode='reflect', normalized=False, onesided=True, name=None)
+```
+
+其中 Paddle 相比 PyTorch 支持更多其他参数，具体如下：
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.stft.md]: 其中 PyTorch 相比 Paddle 支持更多其他参数，具体如下：

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.quantile.md (hunk 1 above)
Comment (co63oc) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.quantile.md]: 已修改

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.quantile.md (hunk 2 above)
Comment (co63oc) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.quantile.md]: 已修改

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.quantile.md (hunk 3 above)
Comment (co63oc) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.quantile.md]: 已修改

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.quantile.md (hunk 4 above)
Comment (co63oc) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.quantile.md]: 已修改

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.size.md (hunk 15 above)
Comment (co63oc) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.size.md]: 已修改

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.scatter.md (hunk 10 above)
Comment (co63oc) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.scatter.md:18]: 测试Tensor.put_along_axis 不使用arr参数 
图片
已修改API文档 https://github.com/PaddlePaddle/docs/pull/5910

**Review by Fancy-hjyp — APPROVED** [review]
LGTM

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.scatter_add.md [hunk 19]
```diff
@@ -0,0 +1,24 @@
...
+### [torch.Tensor.scatter_add](https://pytorch.org/docs/1.13/generated/torch.Tensor.scatter_add.html#torch.Tensor.scatter_add)
+
+```python
+torch.Tensor.scatter_add(dim, index, src)
+```
+
+### [paddle.Tensor.put_along_axis](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/Tensor_cn.html#put-along-axis-arr-index-value-axis-reduce-assign)
+
+```python
+paddle.Tensor.put_along_axis(index, value, axis, reduce="assign")
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.scatter_add.md]: 这两个能对上吗，结果是一样的吗

Comment (co63oc) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.scatter_add.md]: 已修改使用组合 put_along_axis 实现

**Review by zhwesky2010 — APPROVED** [review]
LGTM
